modules/custom/advanced_help_block/library/js/grt-youtube-popup.js

```diff
@@ -0,0 +1,72 @@
+/*!
+ * GRT Youtube Popup - jQuery Plugin
+ * Version: 1.0
+ * Author: GRT107
+ *
+ * Copyright (c) 2017 GRT107
+ * Released under the MIT license
+*/
+
+(function ( $ ) {
+
+    $.fn.grtyoutube = function( options ) {
```

Comment:
Can we add it via composer?
It has MIT LIcense which needs change of OpenY license when code is within repository